src/main/java/org/team1540/bunnybotTank2023/Constants.java

```diff
@@ -1,17 +1,31 @@
 package org.team1540.bunnybotTank2023;
 
-import edu.wpi.first.math.kinematics.DifferentialDriveKinematics;
 import edu.wpi.first.math.util.Units;
 
 public final class Constants {
+    public static final Mode currentMode = Mode.SIM;
```

Comment:
if this is final, when does it get changed to `Mode.REAL`?

Reply:
It's changed manually (yeah it's kinda scuffed)

Reply:
I'd say that's more than just "kinda scuffed," I won't approve this without that fixed. You have a way to tell if the robot is real or not, this variable should only be for choosing between sim and replay.